docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,34 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_. The SDK is a very huge file as it contains tho whole sysroot for the project in addition to some native tools, so depending on the internet connectivity speed it could take a while.
+
+.. note:: Note that the version number at the end of the SDK download file is not the version of the SDK but in fact the version number of poky, which is the upstream Yocto distribution PELUX is using.
+
+Installing the SDK
+------------------
+
+When the SDK has been downloaded from a HTTP server then per default is not executable. This needs to be changed with `chmod`.
+
+.. code-block:: bash
+                
+    cd ~/Downloads
+    chmod +x poky-glibc-x86_64-core-image-pelux-qt-corei7-64-toolchain-2.2.1.sh
+    
+It is a self extracting shell script which can be executed. This will start an interactive program which first asks where to install the SDK. The easiest way is to install it in your `home` directory in it's own directory, like `~/sdk`, instead of the default which is `/opt`. `/opt` is most probably not prepared for your user to install anything into.
```

Comment:
Could you leave a FIXME comment here, pointing to your ticket for fixing default != recommended issue?